README.md

```diff
@@ -162,19 +176,16 @@ Once the new js file has been added, any new projects created will use the updat
         }
       }
```

Comment:
Seems you can't post a suggestion to non-diff code, but the above example should be something like

```js
module.exports = {
    id: 'atari',
    bootstrap: function() {
        require('cordova/channel').onNativeReady.fire();
    }
};
```
which is the file from `osx` with the ID changed: https://github.com/apache/cordova-osx/blob/master/cordova-js-src/platform.js